extensions/wikia/WikiFactory/Metrics/SpecialAWCMetrics.php

```diff
@@ -742,7 +742,7 @@ private function getWikisByNbrPageviews() {
 			$endDate = date( 'Y-m-01', strtotime('now') );
 			$pageviews = DataMartService::getPageviewsMonthly( $startDate, $endDate, $this->cityIds );
 
-			if ( empty( $pageviews ) ) {
+			if ( !empty( $pageviews ) ) {
 				foreach ( $pageviews as $wiki_id => $wiki_data ) {
 					#---
 					if ( $wiki_data['SUM'] > intval($pageViews) ) continue;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)